tests/unit/test_misc.py

```diff
@@ -113,6 +113,16 @@ def test_add_spike_times(self):
         self.assertEqual(ut['spike_times'][0], [0, 1, 2])
         self.assertEqual(ut['spike_times'][1], [3, 4, 5])
 
+    def test_add_waveforms(self):
+        ut = Units()
+        ut.add_unit(waveforms=[[1, 2, 3], [1, 2, 3]])
```

Comment:
Please use floats for the waveform values in both tests. Users are most likely to enter floats. Using floats should also prevent the dtype conversion warning of int -> float on write.

Reply:
@rly Actually, come to think of it, I think users are also likely to enter `int16`, as this is a very common way for acquisition systems to record data. I know that Neuroscope for instance stores spike waveforms in `int16`. I've changed the schema to reflect this. What do you think?

Reply:
@bendichter Isn't unit waveform an average of many spike waveforms?

If you want to change the data type of a column, please do so in another PR.

Reply:
@ajtritt the corresponding schema change is [here](https://github.com/NeurodataWithoutBorders/nwb-schema/pull/448), responding to [this issue](https://github.com/NeurodataWithoutBorders/nwb-schema/issues/248). We would like to add a "waveforms" column to the Units table that holds *all* of the sorted waveforms, not the average.